doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -11,62 +11,168 @@ sort_idx: 30
+* Avoid using weak, vague verbs such as _be_, _have_, _make_, and _do_.
+Such verbs are sometimes necessary, but use a more descriptive verb whenever you can.
+* Use imperative mood for procedures, formulate your instructions as giving commands to the user.
+* Use indicative mood for conveying information such as facts, questions, assertions, or explanations.
+* Omit unnecessary adverbs.
+* Use context to reduce ambiguity.
+Some words can be read as both verbs and nouns, such as _file_, _post_, _input_, etc.
+When using these terms, ensure that the context and sentence structure reduce ambiguity.
+* In general, use second person (pronoun _you_) also known as _direct address_.
+Second person supports a friendly tone because it connects you with the user.
+* Use first person sparingly.
+First person is appropriate when writing from the point of view of the user.
+* Avoid using more than 3 nouns in a noun string as it may be confusing or ambiguous even for native English speakers.
+
+## Text formatting
+* Use **bold** for:
+  * Titles of windows and dialog boxes.
+  * Names of commands, attributes, constants, methods, fields, predefined classes, databases, events, UI elements.
+  * User input.
+* Use _italic_ for:
+  * Technical terms on first mention.
+  * Emphasis.
+  * Placeholders and parameter names.
+* Use links for names of pages and sections.
 * Capitalize only the first word in titles and headings.
-* Do not use capitalization for no apparent reason.
-Use lowercase unless uppercase is justified.
-* Use Heading 4 (i.e. `####`) as the last level heading.
-All other levels are not displayed in table of contents.
-
-## Technical terms and abbreviations
-* Spell out an acronym on its first mention on the page, e.g. *Common Type Library* (CTL).
-* Ensure that an important technical term or abbreviation is included in  [Glossary]({{root_url}}Glossary).
-
-## Font conventions
-* Use **bold** for the titles of windows and dialog boxes; for the names of commands, attributes, constants, methods, fields, predefined classes, databases, events, UI elements; for the user input.
-* Use *italic* for technical terms on first mention and emphasis; for placeholders and parameter names; for section names.
-* Use regular font for other purposes, like the names of user-defined classes, folders, files and file extensions; for key names and combinations; for strings (enclose in quotation marks) and values; for environment variables and error message names; for the names of programs and utilities.
-* Use links for page names.
-
-## Code formatting
-* Format all JSON files with this [online tool](https://jsonformatter.curiousconcept.com/), "4 space tab" profile.
+* Do not use capitalization for no apparent reason. Use lowercase unless uppercase is justified.
+* Use regular text (capitalized where applicable) for all other purposes, such as:
+  * Names of user-defined classes, folders, files and file extensions.
```

Comment:
I'm not clear what "user-defined classes" are doing in this list. Classes are code - no?